pkg/credentialstores/test/image-secrets.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Don't quote me on this, but I think that we prefer yaml over json for things like this.

Reply:
Addressed.